playbooks/openshift-master/private/config.yml

```diff
@@ -88,8 +72,6 @@
 - name: Configure masters
   hosts: oo_masters_to_config
   any_errors_fatal: true
-  vars:
-    openshift_ca_host: "{{ groups.oo_first_master.0 }}"
   pre_tasks:
```

Comment:
pre_tasks and post_tasks could be combined here as well unless there is some reason we want handlers triggered between those stages.  And if that is the case, we probably want to flush handlers at the end of the role anyway.

Reply:
pre_tasks run before roles, post_tasks run after roles.  This play has a lot of roles, I don't think we should combine in this patch.